content/docs/200-what-is-effect/the-effect-type.md

```diff
@@ -0,0 +1,26 @@
+---
+title: The Effect Type
+---
+
+Effect was designed from the ground up to be a completely type-safe representation of any program.
```

Comment:
It would be great to have a short paragraph here about what the benefit is of having a "representation of a program" (i.e. that it can be composed and manipulated in powerful ways to build up complexity through composition instead of by adding complexity to our application logic)